File: DailyProgrammer#295.java
    class ChangeWords {
        public static void changeWords(char[] word1,String word2) {
            System.out.println(word1);
            for(int i = 0; i < word2.length();++i) {
                word1[i] = word2.charAt(i);
                System.out.println(word1);
            }
        }

        public static void main(String[] args) {
            System.out.println("Input Output 1.)");
            changeWords("a fall to the floor".toCharArray(),"braking the door in");
            System.out.println();
            System.out.println();
            System.out.println("Input Output 2.)");
            changeWords("wood".toCharArray(),"book");
            System.out.println();
            System.out.println();
            System.out.println("Input Output 2.)");
            changeWords("floor".toCharArray(),"brake");
            System.out.println();
            System.out.println();
        }
    }

File: DailyProgrammer#295.py
def changeWord(word1, word2) :
    list1 = [letter for letter in word1]
    list2 = [letter for letter in word2]
    print(word1)
    for i in range(0,len(list1)) :
        if list1[i] is not list2[i] :
            list1[i] = list2[i]
            printWord = ''.join(list1)
            print(printWord)



changeWord("floor","break")
print()
print()
changeWord("wood","book")
print()
print()
changeWord("a fall to the floor","braking the door in")
